terraso_backend/apps/graphql/schema/audit_logs.py

```diff
@@ -0,0 +1,53 @@
+import typing as t
+
+from DateTime import DateTime as D
+from graphene import DateTime, List, ObjectType, String
+
+from apps.audit_logs import services
+
+
+class Log(ObjectType):
```

Comment:
Let me explore it more; this scenario is quite an edge because I just realized the story wants a string with a specific format, but it still makes sense. 
It seems to me that maybe I'm trying to eat the cake; all the tooling that works with Dango assumes that It has access to the ORM; if so, it will be hard to enforce clear protocols, as well as typing across Django.  Maybe I should make this a separate issue, use the traditional Django ORM for graphql, and use protocols when consuming logs from a different app.